src/main/java/com/spotify/docker/client/DockerCertificates.java

```diff
@@ -145,8 +145,26 @@ private PrivateKey readPrivateKey(Path file) throws IOException, InvalidKeySpecE
   private static PrivateKey generatePrivateKey(PrivateKeyInfo privateKeyInfo) throws IOException,
           InvalidKeySpecException, NoSuchAlgorithmException {
```

Comment:
You can remove `NoSuchAlgorithmException` now.

Reply:
Done.